opennms-web-api/src/main/java/org/opennms/web/svclayer/support/DefaultGraphResultsService.java

```diff
@@ -383,9 +385,11 @@ public void setEventProxy(EventProxy eventProxy) {
         m_eventProxy = eventProxy;
     }
 
+
     /**
+     * <p>setJsonStore</p>
```

Comment:
Still think this javadoc should be removed since it doesn't really add any value.

Reply:
I prefer consistency with the rest of the code in the file.